src/corehost/packaging/Microsoft.NETCore.App/project.json

```diff
@@ -0,0 +1,46 @@
+{
+  "dependencies": {
+    "Microsoft.CodeAnalysis.CSharp": "1.3.0-beta1-20160321-04",
+    "Microsoft.CodeAnalysis.VisualBasic": "1.3.0-beta1-20160321-04",
+    "Microsoft.CSharp": "4.0.1-rc2-23931",
+    "Microsoft.NETCore.DotNetHostPolicy": "1.0.1-beta-002200",
```

Comment:
I understand this, but with a hardcoded version here I don't understand how we can continuously publish these packages like we do with the others in our build. Or are we creating a new model here?